torch/distributed/rpc/api.py

```diff
@@ -123,7 +123,10 @@ def _wait_all_workers():
     assert (
         _ALL_WORKER_NAMES is not None
     ), "`_ALL_WORKER_NAMES` is not initialized for `def _wait_all_workers`."
-    leader_worker_name = sorted(_ALL_WORKER_NAMES)[0]
+    if exit_on_worker_done is not None:
+        leader_worker_name = _to_worker_info(exit_on_worker_done).name
```

Comment:
Any reason for using the name `exit_on_worker_done`? can we just use `leader`?

Reply:
I'll update this to leader.